source/development/packaging/generate-ova.rst

```diff
@@ -3,60 +3,66 @@
+-  `Vagrant <https://www.vagrantup.com/docs/installation/>`__
+-  `Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+-  `Python <https://www.python.org/download/releases/2.7/>`__
 
- * `Virtual Box <https://www.virtualbox.org/manual/UserManual.html#installation>`_
- * `Vagrant <https://www.vagrantup.com/docs/installation/>`_
- * `Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
- * `Python <https://www.python.org/download/releases/2.7/>`_
+We recommend using a system with at least the following specifications.
 
-Download our wazuh-packages repository from GitHub and go to the ova directory.
+-  8 GB RAM
+-  4 cores CPU
+
+Creating the Wazuh VM
+---------------------
+
+To create the virtual machine follow these steps:
 
-.. code-block:: console
+#. Download our *wazuh-packages* repository from GitHub and navigate to the ``ova/`` directory. Select the version, for example, ``v|WAZUH_CURRENT_OVA|``.
 
- $ git clone https://github.com/wazuh/wazuh-packages && cd wazuh-packages/ova && git checkout v|WAZUH_CURRENT_OVA|
+   .. code-block:: console
 
-Execute the ``generate_ova.sh`` script, with the different options you desire.
+      $ git clone https://github.com/wazuh/wazuh-packages && cd wazuh-packages/ova && git checkout v|WAZUH_CURRENT_OVA|
 
-.. code-block:: console
+#. Execute the ``generate_ova.sh`` script.
 
-  $ ./generate_ova.sh -h
+   -  Using production packages.
 
-.. code-block:: none
-  :class: output
+      .. code-block:: console
 
-  General usage: generate_ova.sh [OPTIONS]
-    -r,    --repository       [Optional] Select the software repository [prod/dev]. By default: prod
-    -s,    --store <path>     [Optional] Set the destination absolute path where the OVA file will be stored.
-    -c,    --checksum         [Optional] Generate checksum [yes/no]. By default: no
-    -g,    --debug            [Optional] Set debug mode on [yes/no]. By default: no
-    -h,    --help             [  Util  ] Show this help.
+         $ ./generate_ova.sh
 
-The options for the repository indicates whether the packages used to install Wazuh are the production ones or not.
+   -  Using development packages.
 
- * prod: The OVA uses released packages.
- * dev: The OVA uses pre-release packages.
+      .. code-block:: console
 
-Below, you will find some examples of how to build a Wazuh virtual machine.
+         $ ./generate_ova.sh -r dev
 
-.. code-block:: console
+   These alternative commands above build a Wazuh VM with Wazuh central components. Check all available options by running the following command.
 
-  # ./generate_ova.sh
+   .. code-block:: console
 
-This will generate a Virtual machine with the Wazuh manager |WAZUH_CURRENT_OVA| and ELK |ELASTICSEARCH_LATEST| installed using production packages.
+      $ ./generate_ova.sh -h
 
-.. code-block:: console
+   .. code-block:: none
+      :class: output
 
-  # ./generate_ova.sh -r dev
+      General usage: generate_ova.sh [OPTIONS]
```

Comment:
The [generate_ova.sh](https://github.com/wazuh/wazuh-packages/blob/v4.5.1/ova/generate_ova.sh) script also includes a `staging` option, we should describe it in the documentation. 
```
DESCRIPTION
        -r,  --repository
                Use development or production repository.
                Values: [prod|dev|staging]. By default: prod.
```

Reply:
The staging option doesn't seem to be useful for the general users.